modules/fedimint-mint-client/src/lib.rs

```diff
@@ -346,19 +346,10 @@ impl FromStr for OOBNotes {
 }
 
 impl Display for OOBNotes {
-    /// Base64 encode a set of e-cash notes for out-of-band spending.
-    ///
-    /// Defaults to standard base64 for backwards compatibility.
-    /// For URL-safe base64 as alternative display use:
-    /// `format!("{:#}", oob_notes)`
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         let bytes = Encodable::consensus_encode_to_vec(self);
 
-        if f.alternate() {
-            f.write_str(&BASE64_URL_SAFE.encode(&bytes))
-        } else {
-            f.write_str(&base64::engine::general_purpose::STANDARD.encode(&bytes))
-        }
+        f.write_str(&BASE64_URL_SAFE.encode(&bytes))
```

Comment:
I guess if you really want the old encoding you can implement it yourself, so we don't even have to keep the alt format.